src/components/input/input.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Grow Elastic TextArea
+ *
+ * Increases the `rows` attribute of a `textarea` until it no longer scrolls.
+ *
+ * @param {HTMLTextAreaElement} textarea - the target `textarea` element
```

Comment:
Can you remove the duplicated type here? When I set up TS I forgot to disable the ESLint rule that requires that the type is in the JSDoc for `.ts` files. I made that change here: https://github.com/cloudfour/cloudfour.com-patterns/pull/726/files#diff-df39304d828831c44a2b9f38cd45289cR35-R38, could you copy those changes into this PR (or I could make a separate PR)?

Reply:
@calebeby I'm a little unclear on the interaction between JSDoc and TS if you're not extracting types from JSDoc.

Normally JSDoc wants you to put the type there. We're not generating JSDoc documentation, so it kinda doesn't matter if we violate the rules, but it seems odd to me.

Is it common to leave the types out of the JSDoc blocks when you're using TS?

Reply:
Yep, it is common! https://github.com/microsoft/tsdoc#what-is-tsdoc